src/Modules/Pages/Controller/PageController.php

```diff
@@ -73,9 +73,17 @@ public function __invoke(Request $request, Revision $revision): Response
             foreach ($revisionBlocks as $revisionBlock) {
                 if ($revisionBlock->getBlock() instanceof Block) {
                     $blockName = (string) $revisionBlock->getBlock();
+
                     if ($this->frontendBlocks->has($blockName)) {
                         /** @var BlockControllerInterface $blockController */
                         $blockController = $this->frontendBlocks->get($blockName);
+
+                        if ($revisionBlock->getBlock()->getBlock()->getModule()->getName() === 'ContentBlocks' &&
```

Comment:
why did you do this? the pages module shouldn't know anything about the content blocks module

Reply:
removed again in  https://github.com/forkcms/forkcms/pull/3555/commits/c24fac573aad3d9e429b9a352c24087b2dd1f862